raiden-dapp/src/store/notifications/getters.ts

```diff
@@ -8,7 +8,9 @@ import { NotificationImportance } from '@/store/notifications/notification-impor
 
 export const getters: GetterTree<NotificationsState, RootState> = {
   notifications: (state: NotificationsState): NotificationPayload[] => {
-    return Object.values(state.notifications).sort((a, b) => a.id - b.id);
+    return Object.values(state.notifications)
+      .sort((a, b) => a.id - b.id)
```

Comment:
Not sure what is more complicated to understand:
- `sort((a, b) => a.id - b.id).reverse()`
- `sort((a, b) => b.id - a.id)`

It doesn't really matter. Just feels like unnecessary work for the CPU and one more line of code. I mean why not sort it directly correct? :thinking:

Reply:
Ah, of course. That slipped my mind. Yes, let's get rid of the reverse function. :)